Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -604,18 +338,48 @@ span.bocListEditableCell .validationErrorMarker[aria-label]:not([title])::after
   box-sizing: border-box;
 }
 
-span.bocListEditableCell .validationErrorMarker[aria-label]:not([title]):focus::after,
-span.bocListEditableCell .validationErrorMarker[aria-label]:not([title]):hover::after
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell .validationErrorMarker[aria-label]:not([title]):focus::after,
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell .validationErrorMarker[aria-label]:not([title]):hover::after
 {
   visibility: visible;
   transition-delay: 0.7s;
 }
+/*endregion*/
 
-span.bocListEditableCell .control > *
+/*region Validation layout*/
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell
 {
-  width: 100%;
-  vertical-align: middle;
+  display: inline-grid;
+  grid-template-columns: auto auto;
+  grid-template-rows: auto auto;
+}
+
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell .control
+{
+  grid-row: 1 / span 2;
+  grid-column: 1 / span 2;
+}
+
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell > .validationErrorMarker
+{
+  grid-row: 1;
+  grid-column: 1;
+  margin: auto var(--remotion-themed-spacing);
+}
+
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell .validationErrorMarker + .control
+{
+  grid-row: 1;
+  grid-column: 2;
+}
+
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell .validationErrorMarker ~ .bocListEditModeValidationMessage
```

Comment:
Now that I understand better, there is a problem with the CSS
The class `bocListEditModeValidationMessage` is customizable via the validator and shouldn't be used for controlling the layout. 

The element with the `bocListEditModeValidationMessage` maybe repeated multiple times, so there can be more than one grow-row-no-2 item in the markup.

Reply:
It's okay if you want to group the validators in an extra div with a dedicated css class.